modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -48,7 +46,7 @@
 
     #
     # Set this to true to configure Liferay CDN as the default repository in the
-    # root project. The default value is false.
+    # root project. The default value is true.
     #
     #liferay.workspace.default.repository.enabled=false
```

Comment:
If the default value is true, then should it be listed as true in the properties file line that is commented out?